mappings/net/minecraft/world/level/ColorResolver.mapping

```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/unmapped/C_qodopjcw net/minecraft/world/level/ColorResolver
```

Comment:
This is mapped in #5, but I see comments about this on said PR, both have no response, so would indicate there is no plan to remove it from #5?

Reply:
I was actually going to remove it here but I forgot